mapcss/mapcss2osmose.py

```diff
@@ -429,8 +431,30 @@ def functionExpression_runtime(t, c):
             "mapcss.round_" if t['name'] == 'round' else
             "mapcss." + t['name']
         )
+
+        if not c.get('flags'):
+            checkValidFunction(t['name'], len(t['params']))
     return t
 
+def checkValidFunction(fn_name, num_params):
+    """
+    Tests if mapcss.* functions actually exist, throws a compile error otherwise
+    """
+    if fn_name[0:7] != "mapcss.":
+        return
+    if not fn_name[7:] in dir(mapcss_lib):
+        raise NotImplementedError("Undefined function '{0}'. Blacklist or implement to avoid runtime errors".format(fn_name))
```

Comment:
Message can be confusing as raised at compile time.

```suggestion
        raise NotImplementedError("Undefined function '{0}'. Blacklist or implement to avoid errors".format(fn_name))
```